### PR TITLE
Video options

### PR DESCRIPTION
**Pull request by Chiragmodi01**
- Video options
- Move to history and clear all
- Move to Watchlater, delete one and clear all
- Playlist modal layout

## [Live](https://video-library-react-git-videooptions-chiragmodi01.vercel.app/)

### CODE REVIEW

**Review by dusariudaykumar — APPROVED** [review]
- The UI looks Great
- Everything looks fine

File: src/comps/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 import './Navbar.css'
 import {useMain} from '../../helpers/context/main-context';
-import {IoIosMenu, IoSearchOutline, CgProfile, BiDotsVerticalRounded, BsMoon, BsSun, IoIosArrowForward, IoSettingsOutline, AiOutlineLogout} from '../../utils/getIcons';
+import {IoIosMenu, IoSearchOutline, CgProfile, BiDotsVerticalRounded, BsMoon, BsSun, IoIosArrowForward, BiMask, AiOutlineLogout, MdFaceRetouchingOff, MdFace} from '../../utils/getIcons';
 import {StyledButton, StyledDropdown} from '../../styledComps/index';
 import { Link } from "react-router-dom";
 import verifyForm from '../../utils/verifyForm';
+import {toast} from 'react-toastify';
 
 function Navbar({toggleDropdownRef}) {
```
Comment (aurobindoGupta) [inline src/comps/Navbar/Navbar.jsx:11]: lets try es6

File: src/comps/Video/Video.css [hunk 2]
```diff
@@ -3,6 +3,7 @@
     gap: .7rem;
     width: 15.85rem;
```
Comment (aurobindoGupta) [inline src/comps/Video/Video.css:4]: this is really specific why?

File: src/pages/Playlist/Playlist.jsx [hunk 3]
```diff
@@ -1,9 +1,42 @@
...
+      <div className="videos-container">
+        {
+        state.playlists.length === 0 &&
+        <div className='empty-page-placeholder flex-centered flex-col gap-4'>
+          <RiPlayList2Fill size="10em" className='page-placeholder icon'/>
+          <h1 className='page-placeholder text'>You've no Playlists</h1>
+        </div>
+        // :
+        // <>
+        // <div className="history-current-day">{currentDate} {currentDay}</div>
```
Comment (aurobindoGupta) [inline src/pages/Playlist/Playlist.jsx:29]: remove commented

**Review by aurobindoGupta — APPROVED** [review]
keep up the good work!!

**Pull request merged**